bin/build-release.sh

```diff
@@ -42,8 +42,8 @@ function clean {
 }
 
 function mvnPackage {
-  mvn -Dcheckstyle.skip=true package
-  mvn -Dcheckstyle.skip=true dependency:copy-dependencies
+  mvn package
```

Comment:
No. This should have been corrected. I will see what happened since my
feature branch which this PR is based on has it removed.

On Thu, Jun 11, 2015 at 2:11 AM, Erik Weathers <notifications@github.com>
wrote:

> In bin/build-release.sh
> <https://github.com/mesos/storm/pull/38#discussion_r32192217>:
>
> > @@ -42,8 +42,8 @@ function clean {
> >  }
> >
> >  function mvnPackage {
> > -  mvn -Dcheckstyle.skip=true package
> > -  mvn -Dcheckstyle.skip=true dependency:copy-dependencies
> > +  mvn package
>
> did we really wanna disable the running of checkstyle here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesos/storm/pull/38/files#r32192217>.
>